honeycomb-template/src/server/routes/health.js

```diff
@@ -1,7 +1,9 @@
 import controller from '../controllers/health';
 
-export default [{
+const routes = [{
```

Comment:
wouldn´t it better, if we name it healthRoutes, instead of routes in every endpoint?